modules/mocks/start.sh

```diff
@@ -1 +0,0 @@
-docker run -p 5555:5555 -v ${PWD}:/tmp/config bbyars/mountebank mb --configfile /tmp/config/mountebank.json
```

Comment:
Please restore this file it is helpful for local development, or please suggest an alternative approach